src/Ocelot/Request/Middleware/DownstreamRequest.cs

```diff
@@ -16,7 +16,7 @@ public DownstreamRequest(HttpRequestMessage request)
         Scheme = _request.RequestUri.Scheme;
         Host = _request.RequestUri.Host;
         Port = _request.RequestUri.Port;
-        AbsolutePath = _request.RequestUri.AbsolutePath;
+        Path = _request.RequestUri.AbsolutePath;
```

Comment:
You rename the property, yet the underlying logic remains. What is the purpose of such a renaming?

Reply:
Because `AbsolutePath` is now a relative path, if the prefix exists (the naming would be misleading).

Reply:
Misleading for whom? I don't think the 'Absolute' prefix is misleading to anyone, as it is an internal class of Ocelot Core. 😉